scripts/plugins/install_plugins.py

```diff
@@ -66,20 +68,20 @@ def Install_Plugins():
                 logger.info("\n~#~#~#~CLONING PLUGIN [{plugin_config}] SUCCESSFULLY COMPLETED~#~#~#~".format(plugin_config=plugin_config))
 
             except git.exc.GitCommandError as exc:
-                logger.info("\n~#~#~#~CLONING PLUGIN [{plugin_config}] FAILED~#~#~#~\n\t{stderr}"
+                logger.error("\n~#~#~#~CLONING PLUGIN [{plugin_config}] FAILED~#~#~#~\n\t{stderr}"
                     .format(
                         plugin_config=plugin_config,
                         stderr=exc
                     ))
-                if BITOPS_fast_fail_mode: quit()
+                sys.exit(1)
```

Comment:
Looks like the `fail_fast` is not really implemented, but it's outside of this PR.

Reply:
If we're going to release v2.0.0 that advertises `fail_fast` functionality (yaml setting, docs https://github.com/bitovi/bitops/blob/plugins/docs/plugins.md#bitopsconfigyaml), but doesn't work yet, - it perhaps better to remove it for now until the full implementation.

How about removing it just from the docs for now so it doesn't break the expectations?  (hidden)